docs/general/metrics.md

```diff
@@ -89,6 +90,28 @@ When introducing a new metric name check all existing schema files to make sure
 the name does not appear as a key of any "rename_metrics" section (keys denote
 old metric names in rename operations).
 
+### Metric attributes
+
+Metric attributes MUST follow the general [attribute naming rules](attribute-naming.md).
```

Comment:
this is stronger than recommendation (SHOULD) in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attribute-naming.md#recommendations-for-opentelemetry-authors

Since attributes are not signal-specific, I think we should stay consistent.

Reply:
Good point. Adapted to SHOULD. Thanks!